### PR TITLE
Fix marquee screen now have scroll view

### PR DESCRIPTION
**Pull request by adids1221**
## Description
Marquee screen now has a scroll view.

## Changelog
Marquee screen now has a scroll view.
The screen does not fit the private demo app.

### CODE REVIEW

File: demo/src/screens/componentScreens/MarqueeScreen.tsx [hunk 1]
```diff
@@ -62,28 +61,32 @@ export default class MarqueeScreen extends Component<{}> {
...
-          {renderMultipleSegmentOptions.call(this, 'Number Of Reps', 'numOfReps', [
-            {label: 'Infinite', value: -1},
-            {label: '1', value: 1},
-            {label: '3', value: 3},
-            {label: '5', value: 5}
-          ])}
-          <View marginV-s2>
-            {renderBooleanOption.call(this, 'Direction Horizontal: Left To Right/Right To Left', 'directionHorizontal')}
-            {renderBooleanOption.call(this, 'Direction Vertical:  Bottom To Up/Up To Bottom', 'directionVertical')}
+        <ScrollView>
```
Comment (lidord-wix) [inline demo/src/screens/componentScreens/MarqueeScreen.tsx:64]: Why not replace the wrapping View with a ScrollView?

Comment (adids1221) [reply demo/src/screens/componentScreens/MarqueeScreen.tsx:64]: You're right I'll just pass the style to the ScrollView

File: demo/src/screens/componentScreens/MarqueeScreen.tsx [hunk 2]
```diff
@@ -61,7 +60,7 @@ export default class MarqueeScreen extends Component<{}> {
 
   render() {
     return (
-      <View flex padding-page>
+      <ScrollView style={{flex: 1, padding: 20}}>
```
Comment (lidord-wix) [inline demo/src/screens/componentScreens/MarqueeScreen.tsx:63]: It is more performant to use react-native's StyleSheet instead of inline style.

**Review by lidord-wix — APPROVED** [review]
Approved.
I left a small comment, it's not so important because it's in the screen, but many users copy our examples in their code so it's better to keep the best practices :)

**Pull request merged**